frontend/.eslintrc.cjs

```diff
@@ -87,6 +87,7 @@ module.exports = {
     'no-shadow': 'off',
     '@typescript-eslint/no-shadow': 'warn',
     'react/prop-types': 'off',
+    'default-param-last': 'off',
```

Comment:
I think it's a good rule to keep to be honest

Reply:
I'd vote keeping it as well

Reply:
Looks like it got a bit more strict and is a bit annoying when we have default params followed by optional params which I find a bit surprising tbh. Anyway I re-added the rules and ignored the errors in-line for now.